client/src/partials/section_resultat/section_resultat.html

```diff
@@ -0,0 +1,194 @@
+  </button>
+</div>
+
+
+<div class="flex-content">
+  <div class="container-fluid">
+    <div class="row">
+      <!-- Complete list of registred Section Resultat -->
+      <div class="col-xs-7">
+        <div class="panel panel-default" style="overflow: auto; max-height: 500px;">
+          <div class="panel-heading">
+            {{ "SECTION_RESULTAT.ALL" | translate }}
+          </div>
+
+          <table class="table table-condensed">
+            <thead>
+              <tr>
+                <th>{{ "COLUMNS.NR" | translate }}</th>
+                <th>{{ "COLUMNS.TEXT" | translate }}</th>
+                <th>{{ "COLUMNS.POSITION" | translate }}</th>
+                <th></th>
+                <th colspan="2">{{ "COLUMNS.ACTION" | translate }}</th>
+              </tr>
+            </thead>
+            <tbody>
+              <tr ng-if="sectionResultatCtrl.session.loading" class="text-center">
+                <td colspan="6"><loading-indicator></loading-indicator></td>
+              </tr>
+              <tr 
+              ng-repeat="sectionResultat in sectionResultatCtrl.sectionResultats | orderBy:'number' | orderBy:'text' " 
+              ng-class="{'rowSelected' : sectionResultatCtrl.sectionResultat.id === sectionResultat.id}" >
+                <td>{{ $index+1 }}</td>
+                <td>{{ sectionResultat.text }}</td>
+                <td>{{sectionResultat.position}}</td>
+                <td>                  
+                  <b ng-show="sectionResultat.is_charge">{{sectionResultatCtrl.doTranslate('SECTION_RESULTAT.CHARGE')}}</b>
+                  <b ng-show="!sectionResultat.is_charge">{{sectionResultatCtrl.doTranslate('SECTION_RESULTAT.PROFIT')}}</b>
+                </td>
+                <td><a class="btn btn-xs btn-default" id="sectionResultat-upd-{{$index+1}}" ng-click="sectionResultatCtrl.update(sectionResultat)"><span class="glyphicon glyphicon-pencil"></span></a></td>
+                <td><a class="btn btn-xs btn-default" id="sectionResultat-del-{{$index+1}}" ng-click="sectionResultatCtrl.del(sectionResultat)"><span class="glyphicon glyphicon-trash"></span></a></td>
+              </tr>
+              <tr ng-if="!sectionResultatCtrl.sectionResultats.length">
+                <td colspan="6">{{ "SECTION_RESULTAT.NO_SECTION_RESULTAT" | translate }}</td>
+              </tr> 
+
+            </tbody>
+          </table>
+        </div>
+      </div>
+
+      <div class="col-xs-5" ng-switch="sectionResultatCtrl.view">
+        <div ng-switch-default>
+          <div class="alert alert-info">
+            <h4>{{ "SECTION_RESULTAT.TITLE" | translate }}</h4>
+            <p>{{ "SECTION_RESULTAT.ALERT" | translate }}</p>
+          </div>
+        </div>
+
+        <!-- Success saving Feed Back -->
+        <div ng-switch-when="create_success">
+          <div class="alert alert-success" id="create_success">
+            <h4>{{ 'SECTION_RESULTAT.SAVE_SUCCESS' | translate }} </h4>
+            </p>
```

Comment:
There are a number of trailing `</p>` tags throughout this template. This is not a blocking issue however it would be good practice to clean these up.